tests/foreman/ui/test_contentview.py

```diff
@@ -644,641 +369,94 @@ def test_positive_update_name(self):
-            version = self.content_views.get_element_value(
-                locators['contentviews.add_cv_version_dropdown'])
-            self.assertEqual(version, 'Always Use Latest (Currently 2.0)')
-
-    @run_in_one_thread
-    @run_only_on('sat')
-    @skip_if_not_set('fake_manifest')
-    @upgrade
-    @tier3
-    def test_positive_promote_with_rh_custom_spin(self):
-        """attempt to promote a content view containing a custom RH
-        spin - i.e., contains filters.
-
-        :id: 7d93c81f-2815-4b0e-b72c-23a902fe34b1
-
-        :setup: Multiple environments for an org; RH content synced
-
-        :expectedresults: Content view can be promoted
-
-        :CaseLevel: System
-        """
-        cv_name = gen_string('alpha')
-        filter_name = gen_string('alpha')
-        rh_repo = {
-            'name': REPOS['rhst7']['name'],
-            'product': PRDS['rhel'],
-            'reposet': REPOSET['rhst7'],
-            'basearch': 'x86_64',
-            'releasever': None,
-        }
-        env_name = gen_string('alpha')
-        # Create new org to import manifest
-        org = entities.Organization().create()
-        with Session(self) as session:
-            make_lifecycle_environment(
-                session, org=org.name, name=env_name)
-            self.assertIsNotNone(session.nav.wait_until_element(
-                locators['content_env.select_name'] % env_name))
-            self.setup_to_create_cv(rh_repo=rh_repo, org_id=org.id)
-            # Create content-view
-            make_contentview(session, org=org.name, name=cv_name)
-            self.assertIsNotNone(self.content_views.search(cv_name))
-            # Add rh repo
-            self.content_views.add_remove_repos(cv_name, [rh_repo['name']])
-            # add a filter
-            self.content_views.add_filter(
-                cv_name,
-                filter_name,
-                FILTER_CONTENT_TYPE['package'],
-                FILTER_TYPE['exclude'],
-            )
-            # assert the added filter visible
-            self.assertIsNotNone(
-                self.content_views.search_filter(cv_name, filter_name))
-            # exclude some package in the created filter
-            self.content_views.add_packages_to_filter(
-                cv_name,
-                filter_name,
-                ['gofer'],
-                ['All Versions'],
-                [None],
-                [None]
-            )
-            self.content_views.publish(cv_name)
-            self.assertIsNotNone(
-                self.content_views.version_search(cv_name, 'Version 1.0'))
-            status = self.content_views.promote(
-                cv_name, 'Version 1.0', env_name)
-            self.assertIn('Promoted to {}'.format(env_name), status)
-
-    @run_in_one_thread
-    @run_only_on('sat')
-    @skip_if_not_set('fake_manifest')
-    @upgrade
-    @tier3
-    def test_positive_publish_with_rh_custom_spin(self):
```

Comment:
it is redundant coverage as it is easy to track any issue with publish in similar test case on promote